Jumpscale/clients/zero_os/ZeroOSClient.py

```diff
@@ -0,0 +1,21 @@
+from Jumpscale import j
+from Jumpscale.sal_zos.node.Node import Node
+
+
+class ZeroOSClient(j.application.JSBaseConfigClass, Node):
+
+    _SCHEMATEXT = """
```

Comment:
I wonder if this is the right place for this schema.
Seems this schema define the config for the protocol client.
For example I tried to create a node like this:
```python
node = j.clients.zos.get('ac1f6b457338',host='10.102.71.171') 
```
Then I realized I also needed to add a password. So I did
```python
node.data.password = '.....'
```
but still I get authentication error when trying to talk to the node. Seems the reason is the protocol client didn't receive the password update in its schema. 
Which leave me with the hack to do
```python
node.client.data.password= '...'
```
Which I believe is wrong. use shouln't have to know about these implementation detail to use the client.

Reply:
It's the same schema defined by zos protocol which is used under the hood, how else would we pass this config? and from where?

the data passed is passed correctly to the client protocol, can you try with j.clients.zos_protocol only?

e.g.  `j.clients.zos.get('ac1f6b457338',host='10.102.71.171')`

Reply:
> It's the same schema defined by zos protocol which is used under the hood, how else would we pass this config? and from where?

Well that's the question I'm also asking.

> j.clients.zos_protocol.get('ac1f6b457338',host='10.102.71.171')

Like I explained in my first comment. This work the first time you create a client. But not when you need to edit the configuration. Just follow the step I describe in my comment and you will see the edited field are not propagated to the zos_protocol client

Reply:
@arahmanhamdy and @serboctor told me the data can be updated, there's something that supports this.

Will try to fix this.

Reply:
Should be resolved by https://github.com/threefoldtech/jumpscaleX/pull/294/commits/15714b393fd54050cfece2780266e9fa98bed40a